origo/sdk/status/status.py

```diff
@@ -1,6 +1,6 @@
 import logging
 
-from origo.sdk import SDK
+from origo.sdk.sdk import SDK
```

Comment:
Should move SDK class up one level so we don't get multiple `.sdk` in module path?

Reply:
I think we can just add `from origo.sdk.sdk import SDK` in the top `__init__.py`, and that should allow `from origo.sdk import SDK` ...

I just got lost reading about what else you can put in `__init__` :grimacing:

Reply:
Agreed that it should be `from origo.sdk import SDK`.